templates/index.html

```
@@ -220,16 +220,20 @@ <h1 class="text-3xl font-extrabold tracking-tight text-gray-900 dark:text-white"
           <!-- View 2: Anki Search -->
-          <div id="mediaAnkiSearchView" inert class="absolute inset-0 p-4 md:p-6 overflow-y-auto flex flex-col gap-4 w-full transform translate-x-full transition-transform duration-300 bg-white dark:bg-gray-800 invisible">
-            <h3 class="text-xl font-bold dark:text-gray-100">Update Specific Anki Card</h3>
-            <p class="text-sm text-gray-600 dark:text-gray-400">Enter the Note ID (NID) of the card you want to update. You can find this in Anki's browser.</p>
-            <div class="flex gap-2">
-              <input type="number" id="ankiTargetNid" aria-label="Note ID" placeholder="e.g. 1700000000000" class="flex-1 p-2 border rounded dark:bg-gray-800 dark:border-gray-600 dark:text-white focus:outline-none focus:ring-2 focus:ring-indigo-500 transition">
-              <button onclick="sendToAnkiSpecific(this)" class="flex items-center justify-center gap-2 w-36 bg-indigo-600 text-white px-4 py-2 rounded font-bold hover:bg-indigo-700 transition shadow-sm whitespace-nowrap">Update Card</button>
+          <div id="mediaAnkiSearchView" inert class="absolute inset-0 p-4 md:p-6 flex flex-col w-full transform translate-x-full transition-transform duration-300 bg-white dark:bg-gray-800 invisible">
+            <!-- Raycast-style Search Header -->
+            <div class="flex items-center border-b dark:border-gray-700 shrink-0 pb-4 mb-4 gap-3 relative">
+              <div class="relative flex-1">
+                 <input type="text" id="ankiCardSearchInput" placeholder="Search your Anki collection..." class="w-full text-xl p-2 bg-transparent border-none focus:ring-0 dark:text-white placeholder-gray-400 font-medium" oninput="debounceSearchAnkiCards()" onkeydown="if(event.key === 'Escape') toggleModalView('mediaExtractView')">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)